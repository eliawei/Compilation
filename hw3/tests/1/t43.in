//else far from while

void main()
{
    int x = 4;
    byte y = 100 b;
	while(true)
		int z = 2;
	int a;
	else {
		int b;
	}
}