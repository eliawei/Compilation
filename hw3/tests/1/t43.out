---end scope---
z INT 2
line 10: syntax error
